src/quality-menu-button.js

```diff
@@ -70,20 +70,13 @@ class QualityMenuButton extends MenuButton {
     this.hide = this.hide.bind(this);
 
     this.handleQualityChange_ = this.handleQualityChange_.bind(this);
-    this.changeHandler_ = () => {
-      const defaultResolution = this.options_.defaultResolution;
-
-      for (let i = 0; i < this.items.length; i++) {
-        if (this.items[i].options_.label.indexOf(defaultResolution) !== -1) {
-          this.items[i].handleClick();
-        }
-      }
-    };
+    this.firstChangeHandler_ = this.firstChangeHandler_.bind(this);
+    this.enableDefaultResolution_ = this.enableDefaultResolution_.bind(this);
 
     this.on(this.qualityLevels_, 'addqualitylevel', this.update);
     this.on(this.qualityLevels_, 'removequalitylevel', this.update);
     this.on(this.qualityLevels_, 'change', this.handleQualityChange_);
-    this.one(this.qualityLevels_, 'change', this.changeHandler_);
+    this.one(this.qualityLevels_, 'change', this.firstChangeHandler_);
```

Comment:
could you please add TODO to remove this option?

Reply:
Done